hono/migrate.ts

```diff
@@ -2,12 +2,14 @@ import { config } from 'dotenv'
 import { drizzle } from 'drizzle-orm/postgres-js'
 import { migrate } from 'drizzle-orm/postgres-js/migrator'
 import postgres from 'postgres'
+import { itemsExtended } from './src/schema'
 
 config({ path: '.dev.vars' })
 
 const main = async () => {
   const db = drizzle(postgres(process.env.DATABASE_URL!, { ssl: 'require', max: 1 }))
```

Comment:
Should max connections be passed in as an ENV var? Could help with auto scaling.

Reply:
`max` is explicitly set to 1 since the purpose of this script is to run db migrations (which should be done in a sync/non-concurrent manner).